elastic/kibana
# [RAM] Use bulk enable and disable everywhere in UI

Close: https://github.com/elastic/kibana/issues/147099

## Summary

Previously _bulk_enable and _bulk_disable were created. I called it from UI, but from limited places.
Here in this PR I want to completely replace old single enable and disable with new bulk API. 

## How to test

Need to go to each place where it's possible to enable/disable rule and check if _bulk_disable/_bulk_enable was called.
I would appreciate it, I could miss some.

### Checklist
